changelogs/unreleased/2443-improve-relation-depracation-msg.yml

```
@@ -0,0 +1,4 @@
+description: Change the relation deprecation to be more accurate.
```

I'd add a feature section. It's a very small improvement but an improvement but an improvement nonetheless.
```suggestion
description: Change the relation deprecation warning to be more accurate.
```